index.html

```diff
@@ -0,0 +1,216 @@
+                  font-size: 24px;
+                  font-weight: 500;
+                  line-height: 32px;
+                  letter-spacing: 0.05em;
+                  margin-top: 24px;
+                "
+              >
+                가장 HOT한 중고거래 물품을<br />판다마켓에서 확인해 보세요
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="productInfo">
+          <div>
+            <div>
+              <h2
+                style="
+                  color: #3692ff;
+                  font-size: 18px;
+                  font-weight: 700px;
+                  line-height: 26px;
+                  margin-bottom: 12px;
+                "
+              >
+                Search
+              </h2>
+              <h1>구매를 원하는<br />상품을 검색하세요</h1>
+              <p
+                style="
+                  font-size: 24px;
+                  font-weight: 500;
+                  line-height: 32px;
+                  letter-spacing: 0.05em;
+                  margin-top: 24px;
+                "
+              >
+                구매하고 싶은 물품은 검색해서<br />쉽게 찾아보세요
+              </p>
+            </div>
+          </div>
+          <img
+            src="images/home/search-image.png"
+            alt="구매 상품 검색"
+            width="50%"
+          />
+        </div>
+
+        <div class="productInfo">
+          <img
+            src="images/home/register-image.png"
+            alt="판매 상품 등록"
+            width="50% "
+          />
+          <div>
+            <div>
+              <h2
+                style="
+                  color: #3692ff;
+                  font-size: 18px;
+                  font-weight: 700px;
+                  line-height: 26px;
+                  margin-bottom: 12px;
+                "
+              >
+                Register
+              </h2>
+              <h1>판매를 원하는<br />상품을 등록하세요</h1>
+              <p
+                style="
+                  font-size: 24px;
+                  font-weight: 500;
+                  line-height: 32px;
+                  letter-spacing: 0.05em;
+                  margin-top: 24px;
+                "
+              >
+                어떤 물건이든 판매하고 싶은 상품을<br />쉽게 등록하세요
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section id="bottomBanner" class="bannerImage">
+        <div class="wrapper">
+          <h1>
+            믿을 수 있는 <br />
+            판다마켓 중고거래
+          </h1>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <div>@codeit - 2024</div>
+      <div id="footerMeun">
+        <a href="privacy.html">Privacy Policy</a>
+        <a href="faq.html">FAQ</a>
+      </div>
+
+      <div id="social">
+        <a
+          href="https://www.facebook.com/"
+          target="_blank"
+          rel="noopener noreferrer"
+          ><img
+            src="images/socialIcon/facebook-logo.svg"
+            alt="facebook"
+            width="20"
+        /></a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
      <ul id="social">
        <li>
          <a
            href="https://www.facebook.com/"
            target="_blank"
            rel="noopener noreferrer">
            <img
              src="images/socialIcon/facebook-logo.svg"
              alt="facebook"
              width="20"/>
          </a>
        </li>
```